next/components/forms/segments/AccountSectionHeader/TaxFeeSectionHeader.tsx

```diff
@@ -0,0 +1,115 @@
+import FileDownload from '@assets/images/account/file_download.svg'
+import PaymentIcon from '@assets/images/account/payment-icon.svg'
+import { useRouter } from 'next/router'
+import { useTranslation } from 'next-i18next'
+
+import Button from '../../simple-components/Button'
+
+type AccountSectionHeaderBase = {
+  title: string
+}
+
+const TaxFeeSectionHeader = (props: AccountSectionHeaderBase) => {
+  const { t } = useTranslation('account')
+  const router = useRouter()
+  return (
+    <div className="flex flex-col items-start bg-gray-50 sm:px-28 sm:py-6 p-4 mt-16 lg:mt-28 gap-4">
+      <div className="flex items-center gap-0.5 cursor-pointer">
+        <svg
+          width="20"
+          height="20"
+          viewBox="0 0 20 20"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M12.8462 6.17917L11.667 5L6.66699 10L11.667 15L12.8462 13.8208L9.02533 10L12.8462 6.17917Z"
+            fill="#333333"
+          />
+        </svg>
+        <button
+          type="button"
+          className="underline text-p-sm"
+          onClick={() => router.push('/account/taxes-and-fees')}
+        >
+          {t('back_to_list')}
+        </button>
+      </div>
+      <div className="flex flex-col items-start gap-2 w-full h-full">
+        <div className="text-p-base text-main-700 font-semibold sm:block hidden">
```

Comment:
please use classes from typography.css instead of setting sizes and weights manually - get in touch with @AntonZezul if you need more info.